rest/src/main/java/org/jboss/pnc/rest/trigger/BuildTriggerer.java

```diff
@@ -144,4 +170,56 @@ private BuildConfigurationAudited getLatestAuditedBuildConfiguration(Integer bui
         }
         return buildConfigRevs.get(0);
     }
+    
+    /*
+     * TODO: Do not ignore certificates, rather setup servers properly.
+     */
+    private void signalBpmEvent(String uri) {
+        if (bpmConfig == null) {
+            try {
+                bpmConfig = new Configuration().getModuleConfig(BpmModuleConfig.class);
+            } catch (ConfigurationParseException e) {
+                log.error("Error parsing BPM config.", e);
+            }
+        }
+        
+        SSLContextBuilder builder = new SSLContextBuilder();
+        try {
+            builder.loadTrustMaterial(null, new TrustStrategy() {
+                @Override
+                public boolean isTrusted(X509Certificate[] chain, String authType) throws CertificateException {
+                    return true;
+                }
+            });
+        } catch (NoSuchAlgorithmException | KeyStoreException e1) {
+            e1.printStackTrace();
+        }
+
+        SSLConnectionSocketFactory sslSF = null;
+        try {
+            sslSF = new SSLConnectionSocketFactory(builder.build(), SSLConnectionSocketFactory.ALLOW_ALL_HOSTNAME_VERIFIER);
+        } catch (KeyManagementException | NoSuchAlgorithmException e1) {
+            e1.printStackTrace();
+        }
+
+        CloseableHttpClient httpclient = HttpClients.custom().setSSLSocketFactory(sslSF)
+                .setHostnameVerifier(new AllowAllHostnameVerifier()).build();
+
+        HttpPost request = new HttpPost(uri);
+        request.addHeader("Authorization", getAuthHeader());
+        log.info("Executing request " + request.getRequestLine());
+
+        try (CloseableHttpResponse response = httpclient.execute(request)) {
+            log.info(response.getStatusLine());
+            response.close();
+            httpclient.close();
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
The retries could be implemented in the extracted httpClient class as an optional parameter I guess. WDYT?
Sorry, my fault, I will fix the printStackTrace.